### ISSUE TITLE
Definition for rule '@typescript-eslint/no-redeclare' was not found  @typescript-eslint/no-redeclare

### ISSUE THREAD

**Issue by evsar3** [standard/eslint-config-standard-with-typescript#450]
**What version of this package are you using?**
19.0.1

**What operating system, Node.js, and npm version?**
Windows 10 v2004, Node.js v13.3.0, npm v6.13.1

**What happened?**
All my TS files get an error on the very first character of the file:
*Definition for rule '@typescript-eslint/no-redeclare' was not found  @typescript-eslint/no-redeclare*

**What did you expect to happen?**
No error should be shown.

**Are you willing to submit a pull request to fix this bug?**
No. Unfortunately I don't have the level of expertise to contribute with this project yet.

**Extra information**
dummy.ts
````typescript
import * as fs from 'fs'

fs.readFileSync('./dummmy.txt')
````
.eslintrc.js
````javascript
module.exports = {
  extends: 'standard-with-typescript',
  parserOptions: {
    project: './tsconfig.json'
  }
}
````
.eslintignore
````
node_modules
dist
.eslintrc.js
````
tsconfig.json
````json
{
  "compilerOptions": {
    "target": "ES6",
    "module": "CommonJS",
    "outDir": "./dist",
    "rootDir": "./src",
    "strict": true,
    "esModuleInterop": true,
    "skipLibCheck": true,
    "forceConsistentCasingInFileNames": true
  }
}
````
package.json (devDependencies)
````
"devDependencies": {
    "@types/node": "^14.11.5",
    "@types/ws": "^7.2.7",
    "@typescript-eslint/eslint-plugin": "^4.4.0",
    "eslint": "^7.11.0",
    "eslint-config-standard-with-typescript": "^19.0.1",
    "eslint-plugin-import": "^2.22.1",
    "eslint-plugin-node": "^11.1.0",
    "eslint-plugin-promise": "^4.2.1",
    "eslint-plugin-standard": "^4.0.1",
    "lite-server": "^2.5.4",
    "ts-node-dev": "^1.0.0-pre.63",
    "typescript": "^4.0.3"
}
````

On linting 'dummy.ts' the error must be shown in the very first character of the file:
````
<file-path>\dummy.ts
  1:1  error  Definition for rule '@typescript-eslint/no-redeclare' was not found  @typescript-eslint/no-redeclare
````

**Comment by evsar3** [comment]
Right after open this issue I realized that I was using a wrong version of `@typescript-eslint/eslint-plugin`. In order to solve the issue, I just upgraded it's version to `^4.4.0`.

**Comment by evsar3** [comment]
This is possible related to this commit https://github.com/standard/eslint-config-standard-with-typescript/commit/73e39a6f96df99895af67ddb77885adff3bd7cff which was merged in this PR #410

**Comment by zhuqingguang** [comment]
[downgrade to v19.0.0](https://github.com/standard/eslint-config-standard-with-typescript/pull/410#issuecomment-687140787) is working

**Comment by mightyiam** [comment]
Seems like this had been fixed in #410.

**Comment by mightyiam** [comment]
Oh, I suppose not. Anyway, if this is still an issue, please reopen.
